builtin_metadata.json

```diff
@@ -17382,6 +17383,31 @@
     },
     "wasm": false
   },
+  "strings.count": {
+    "args": [
+      {
+        "description": "string to search in",
+        "name": "cheese",
+        "type": "string"
+      },
+      {
+        "description": "substring to look for",
+        "name": "e",
+        "type": "string"
+      }
```

Comment:
@Manish-Giri the names here are probably from a previous iteration. I think a `make generate` should take care of this.

Reply:
@ashutosh-narkar Updated